SimTKcommon/include/SimTKcommon/internal/ClonePtr.h

```diff
@@ -202,7 +207,11 @@ template <class T> class ClonePtr {
         other.reset(p);
         reset(otherp);
     }
-     
+
+    DEPRECATED_14("use get() instead")
+    const T* getPtr()  const  { return get(); }
+    DEPRECATED_14("use upd() instead")
+    T* updPtr() { return upd(); }
```

Comment:
> What would you think about instead hiding deprecated things from doxygen? Is there a good reason to keep documenting them if we don't want people to use them?

I think hiding a method from doxygen, if it exists, can be very frustrating for users. For many, I think doxygen represents the API. If they can use a method but can't find evidence that it exists in the library  (assuming they are unwilling to look at the source code), they might get very confused, re-download Simbody, reinstall it, etc.

Getting rid of macros is nice, especially for `OVERRIDE_11`, which did not add additional functionality. However, Simbody does uses macros in many places where they *are* useful, and I think this could be such a situation.

I still prefer using the macro to insert doxygen documentation (I don't like public methods without doxygen documentation and I don't like hiding methods from doxygen). Perhaps there could be multiple variants: a temporary `DEPRECATED_14` macro that intends to hold us over until c++14 support, and a `SimTK_DEPRECATED` that will stick around and that inserts doxygen documentation.

I'm also probably biased by my excitement in being able to insert doxygen documentation via macros for the OpenSim Properties. Anyway, having the DEPRECATED_14 macro is an improvement over the previous situation.

By the way, Doxygen has a way to mark a method as deprecated: http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmddeprecated.

Ideally, a future version of doxygen would just pick up the C++14 deprecated attribute and insert documentation for it.